linode/manager
# feat: [M3-8172] - Add CopyableTextField story and clean up components

## Description 📝
Add CopyableTextField story and merge `CopyableAndDownloadableTextField` component into `CopyableTextField`

## Preview 📷
[<video src="https://github.com/user-attachments/assets/1631f258-fab4-479a-8094-599dec4e230a" />](https://github.com/user-attachments/assets/1631f258-fab4-479a-8094-599dec4e230a)

## How to test 🧪
### Verification steps
(How to verify changes)
- Run storybook locally and verify changes
- There should be no regressions in the app where `CopyableTextField` is used

## As an Author I have considered 🤔

*Check all that apply*
